static/bidder-info/limelightDigital.yaml

```diff
@@ -1,4 +1,4 @@
-endpoint: "http://{{.Host}}.ortb.net/openrtb/{{.PublisherID}}"
+endpoint: "http://ads-pbs.ortb.net/openrtb/{{.PublisherID}}?host={{.Host}}"
```

Comment:
similar to https://github.com/prebid/prebid-server/pull/2653/files#r1143367385, change doesn't seems specific to adding evolution technologies adapter